### ISSUE TITLE
git@github.com remote url

### ISSUE THREAD

**Issue by louisponet** [Roger-luo/Ion#40]
Hi,

Super cool tool, thanks!

Most of my repos use a remote url defined in the config like 
`git@github.com:louisponet/RemoteHPC.jl`
rather than the actual https::/ url.

This leads to `Invalid remote url` errors.

Is it possible to handle this case?

cheers

**Comment by Roger-luo** [comment]
Yes! I thought I handled this case, but it is indeed not tested... (only 55% at the moment), let me check if I can fix it a bit later this week

**Comment by Krastanov** [comment]
maybe redundant, but just mentioning `git@github.com` urls still seems to be an issue on the current version

**Comment by Roger-luo** [comment]
yeah sorry, I haven't had time working on this
